Tools/jit/ignore-tests-emulated-linux.txt

```diff
@@ -82,3 +82,4 @@ test.test_subprocess.ProcessTestCaseNoPoll.test_empty_env
 test.test_subprocess.ProcessTestCaseNoPoll.test_file_not_found_includes_filename
 test.test_subprocess.ProcessTestCaseNoPoll.test_one_environment_variable
 test.test_venv.BasicTest.test_zippath_from_non_installed_posix
+test.test_subprocess.POSIXProcessTestCase.test_vfork_used_when_expected
```

Comment:
Minor, can you keep this list alphabetized? This should go right after `test.test_subprocess.POSIXProcessTestCase.test_exception_bad_executable`.

Reply:
Done